src/plugins/intel_gpu/tests/fusions/convolution_fusion_test.cpp

```diff
@@ -2564,11 +2602,11 @@ TEST_P(conv_fp32_reorder_bfyx_to_fsv32_conv_basic, basic) {
     execute(p);
 }
 INSTANTIATE_TEST_SUITE_P(fusings_gpu, conv_fp32_reorder_bfyx_to_fsv32_conv_basic, ::testing::ValuesIn(std::vector<convolution_test_params>{
-    convolution_test_params{ FSV32_CASE_CONV_FP32_1,  3, 3 }
+    convolution_test_params{ FSV32_CASE_CONV_FP32_1,  3, 4 }
 }));
 
-
-class conv_fp32_reorder_bfyx_to_fsv32_conv_mean : public ConvFusingTest {};
+// 'reorder_fsv32' should NOT be optimized because it has 'mean'
```

Comment:
I'd like to suggest to mention in a ligher way, e.g.,  "'reorder_fsv32' is not being fused in the current impl, since it has 'mean'.

Reply:
applied